scripts/start.sh

```
@@ -0,0 +1,44 @@
+source "$NVM_DIR/nvm.sh"
+
+# --- Use Node Version ---
+cd "$FRONTEND_DIR"
+NODE_VERSION=$(cat .nvmrc 2>/dev/null || echo "22")  # Default to Node.js v22 if .nvmrc is not found
+echo "▶ Using Node.js v$NODE_VERSION"
+nvm use $NODE_VERSION || nvm install $NODE_VERSION
+
+# --- Load .env if available ---
+if [ -f "$FRONTEND_DIR/.env" ]; then
+  echo "🌱 Loading .env file..."
+  export $(grep -v '^#' "$FRONTEND_DIR/.env" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)